### ISSUE TITLE
[Question] Guidelines for design choices

### ISSUE THREAD

**Issue by Bernardo-Favoreto** [facebookresearch/wav2letter#780]
### Question
How should I guide my design choices (AM architecture, hyperparameters) for training?
[A clear, concise description of your setup and question]

#### Additional Context
I am trying to train and end-to-end ASR system on custom data (which is in Portuguese - BR) and would like to know how should one go about choosing the appropriate architecture/training parameters? I know this may be a bit too broad, but I believe there might exist some guidelines and general principles. 
For instance, the SOTA architectures seems like a good baseline, but I don't have as many data as benchmark datasets. Should I simply reduce their layers? And how would this affect hyperparameters? I have tried to reduce the given transformer architecture, but the model seemed to be stuck at 50+ loss after around 30 epochs. For elucidation, my data preparation was done basically following the tutorial for librispeech. I have my 3 lists file (train, dev, test), my lexicon.txt and tokens.txt (generated from the set of characters of my dataset) and this is what I have been using so far. 
Also, if I could have an estimation of how long would take to train said model in ~100 hours of speech in a single GPU?

Please feel free to ask for more information.

Thanks in advance.
